app/assets/javascripts/modules/WizardForm.js

```diff
@@ -0,0 +1,61 @@
+define(['jquery'], function($) {
+  'use strict';
+
+  var $form = $('[data-wizard-form] form'),
+      $formItems = $form.find('.wizard-form-item'),
+      $submitButton = $form.find('button.button--primary'),
+      $prevButton = $form.find('button.button--secondary'),
+      $currentStep = $formItems.first(),
+      $currentStepNumber = 0;
+
+  $formItems.hide().first().show().addClass('show');
+  $form.addClass('js-wizard');
```

Comment:
We don't need to add this class here if we use the `js` hook in the `html` element.

Reply:
Removed